server/api/editApplicantProfile.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const router = express.Router();
+const {
+  editApplicantProfile
+} = require("../services/database/editApplicantProfile");
+const {
+  deleteApplicantSkillsForApplicantByapplicantProfileID
+} = require("../services/database/applicantSkills");
+const { newApplicantSkills } = require("../services/database/applicantSkills");
+
+router.put("/:id", (req, res) => {
+  const { id } = req.params;
+  const { name, about, city, cvLink, skills, rightToWork } = req.body;
```

Comment:
I like this some improvements here using Destructuring 👍

Reply:
thanks